capture/generate/qrandom.py

```diff
@@ -361,6 +360,19 @@ def wolfram_sampling(expoverview, rdict, vollimits, rxndict, wellnum, userlimits
     experiment_mmol_df = pd.concat([experiment_mmol_df, portion_mmol_df], axis=1)
     experiment_df = pd.concat([experiment_df, portion_df], axis=1)
 
+    portionnum +=1
+    if portionnum < len(expoverview):
+        prdf, prmmoldf, version = default_sampling(expoverview,
+                                           rdict,
+                                           vollimits,
+                                           rxndict,
+                                           wellnum,
+                                           userlimits,
+                                           experiment,
+                                           portionnum=portionnum)
```

Comment:
I see what you did with the `portionnum`s here, very clever. 

Two questions: 
1. Do you want me to rewrite this so that we can use the mathematica sampler for each portion? That's actually how I originally wrote the mathematica integration before Josh shot down the portions, so it wouldn't take much. 
2. Any reason this is only coded for 2 portions? Why not just loop over all of them, however many there may be.

Reply:
I didn't realize it was written for two portions, but that is still enough for now.  

I think that Josh's input is good, but it shouldn't stop development moving forward. If we make a mistake then that is on us, but integrating mathematica for portions would have avoided this headache. 

I think this is fine for now.  No need to over engineer the hack.  The better use of time would be porting the mathematica to python.

Reply:
Ahh, I see how that works.  The argument `portion_num` in the `default_sampler` controls which portion to start looping from. I request calling the parameter `start_portion_idx` to make that more clear. For some reason the parameter name `portion_num` made me think that `default_sampler` only operated on one portion at a time, despite my having read the function earlier and noticed the while loop over the portions.